gtsam/inference/Factor.h

```diff
@@ -163,12 +173,15 @@ typedef FastSet<FactorIndex> FactorIndexSet;
     /** Serialization function */
     friend class boost::serialization::access;
     template<class Archive>
-    void serialize(Archive & ar, const unsigned int /*version*/) {
+    void serialize(Archive & ar, const unsigned int version) {
       ar & BOOST_SERIALIZATION_NVP(keys_);
+      if (version>=1) ar & BOOST_SERIALIZATION_NVP(active_);
```

Comment:
Probably need to check other places for latest version number?

Reply:
I think "version" is local to each class, so it's only required to update it for the Factor base class... 
https://theboostcpplibraries.com/boost.serialization-archive